File: BountyHuntersAPI/BountyHuntersAPI/Models/Tournament.cs
using System.Collections.Generic;
using   System.ComponentModel.DataAnnotations;
namespace BountyHuntersAPI.Models
{
    public class Tournament
    {
        [Key]
        public int TournamentId { get; set; }
        public string Name { get; set; }
        public int MatchesCount { get; set; }
        public int CurrentMatch { get; set; }
        public int NumberGoals { get; set; }
        public int PlayersInTeam { get; set; }
        public int PlayersInKnockout { get; set; }
        public int GoalsToWin { get; set; }
        public ICollection<Player> Players { get; set; } = new List<Player>();
        public ICollection<Match> Matches { get; set; } = new List<Match>();
    }
}

File: bounty-hunters/src/app/Models/Tournament.ts
import { Player } from './Player';
import { Match } from './Match';

export class Tournament {
  tournamentId: number;
  name: string;
  players: Player[];
  matches: Match[];
  matchesCount: number;
  currentMatch: number;
  playersInKnockout: number;
  playersInTeam: number;
  goalsToWin: number;
}
